automapper/src/main/java/com/valqueries/automapper/SqlDialect.java

```diff
@@ -113,6 +114,48 @@ default String getSqlType(Class type, Property property) {
 
 	String limit(int offset, Integer limit);
 
+	String update(TypeDescriber<?> typeDescriber, List<Element> elements, List<Property.PropertyValue> newPropertyValues);
+
+	default String describe(String tablename) {
+		return "DESCRIBE " + tablename + "";
+	}
+
+	default String describeIndex(String tablename) {
+		return "show index from " + tablename + "";
+	}
+
+	default String getDescribedFieldColumnName() {
+		return "Field";
+	}
+
+	default String getDescribedFieldColumnType() {
+		return "Type";
+	}
+
+	default SqlDescriber.DbRow getDbRow(OrmResultSet ormResultSet) {
+		try {
+			return new SqlDescriber.DbRow(ormResultSet.getString("Field"), ormResultSet.getString("Type"), ormResultSet.getString("Null").equals("Yes") ? true : false);
```

Comment:
should this not call `getDescribedFieldColumnName()` rather than hard-coded "Field"

Reply:
134c57f